Makefile

```diff
@@ -102,31 +127,50 @@ generated-code: $(OUTPUT_DIR)/.generated-code update-licenses
 SUBDIRS:=pkg test
 $(OUTPUT_DIR)/.generated-code:
 	mkdir -p ${OUTPUT_DIR}
-	rm -rf vendor_any
```

Comment:
since this is now moved to `make clean`, should this target be updated to call `clean`?

Reply:
Good call! Will update